pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -394,6 +394,10 @@ func (gce *GCECloud) EnsureTCPLoadBalancer(name, region string, externalIP net.I
 		PortRange:  fmt.Sprintf("%d-%d", minPort, maxPort),
 		Target:     gce.targetPoolURL(name, region),
 	}
+	if externalIP != nil {
```

Comment:
We don't use the externalIP field from the kubectl command. we use the load-balancer-ip field.
Here the variable name is called externalIP, but I renamed it to loadBalancerIP to be more clear.